MastodonSDK/Sources/MastodonSDK/Entity/Mastodon+Entity+Instance.swift

```diff
@@ -114,7 +114,7 @@ extension Mastodon.Entity.Instance {
 }
 
 extension Mastodon.Entity.Instance {
-    public struct Configuration: Codable {
+    public struct Configuration: Codable, StatusesMediaAttachmentsPollsContaining {
```

Comment:
What a protocol-name oO Only `Mastodon.Entity.[V2].Instance` implement it, right?
What do you think about `SomethingSomethingInstanceConfiguration`?

Reply:
Yeah I'm also torn about this one 😄  Let me quickly think about it.